kubernetes-client/src/main/java/io/fabric8/kubernetes/client/URLFromNodePortImpl.java

```diff
@@ -44,7 +44,7 @@ public String getURL(Service service, String portName, String namespace, Kuberne
           }
         }
       } catch (KubernetesClientException exception) {
-        logger.warn("Could not find a node! " + exception);
+        logger.warn("Could not find a node! {0}", exception);
```

Comment:
I think it would be better if we log log.warn(String, Throwable) method:
[0] http://www.slf4j.org/apidocs/org/slf4j/Logger.html#warn(java.lang.String,java.lang.Throwable)
```suggestion
        logger.warn("Could not find a node!", exception);
```

Reply:
yeah this make sense